tests/Fixtures/TestBundle/Serializer/Denormalizer/DiscrAbstractDummyDenormalizer.php

```diff
@@ -0,0 +1,66 @@
+<?php
```

Comment:
IMO `abstract` classes are a bad idea when dealing with API Platform, and with Doctrine generally speaking.

However this specific case should be handle automatically, we even have a test for that: https://github.com/api-platform/core/blob/master/features/main/crud_abstract.feature